### PR TITLE
chore: add revert tx debug script

### PR DESCRIPTION
**Pull request by krtk6160**

### CODE REVIEW

File: src/debug/revert-tx.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ *	. ./.envrc && yarn ts-node \
+ *		--files \
+ *			-r tsconfig-paths/register \
+ *			-r src/services/tracing.ts \
+ *		src/debug/revert-tx.ts <txId>
+ */
+
+import { isUp } from "@services/lnd/health"
+import { lndsConnect } from "@services/lnd/auth"
+
```
Comment (vindard) [inline src/debug/revert-tx.ts:12]:
```suggestion
```

File: src/debug/revert-tx.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  const journalId = tx._journal.toString()
+
+  await MainBook.void(journalId, "Buggy payment")
+
+  return true
+}
+
+setupMongoConnection(false)
+  .then(async (mongoose) => {
+    await Promise.all(lndsConnect.map((lndParams) => isUp(lndParams)))
```
Comment (vindard) [inline src/debug/revert-tx.ts:34]:
```suggestion
```

**Review by vindard — CHANGES_REQUESTED** [review]
Tested locally, looks good to me for removing intraledger transactions. I don't think we'd need the lnd-related methods since we're only doing ledger operations here.

Do we plan to use this script for any transaction types other than intraledger ones?

**Comment by dolcalmi** [comment]
ummm this script is too dangerous... we need to have a script for specific cases with logic that guarantees at least basic business rules like `src/debug/void-payment.ts`

**Comment by dolcalmi** [comment]
we can reopen it if it is necessary

**Pull request closed without merging**